api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -301,6 +301,32 @@ message HttpConnectionManager {
     type.http.v3.PathTransformation http_filter_transformation = 2;
   }
 
+  message ProxyStatusConfig {
+    enum ProxyName {
+      // Use the string "envoy".
```

Comment:
nit: s/envoy/Envoy/

Reply:
I am afraid this would be inconsistent with the default value of server_name above, which is `envoy` and not `Envoy`. Thoughts?

Reply:
Thanks, sounds good.